scanner/hack/quay/main.go

```diff
@@ -0,0 +1,117 @@
+package main
```

Comment:
nit: would these `scanner/hack/*` files be better positioned if moved to a `test`, `testutils`, etc. directory?

Reply:
I've found `hack` as a pretty common directory in open source codebases (usually ones written in Go...). For example https://pkg.go.dev/k8s.io/kubernetes/hack

It's meant to capture some one-off scripts and such. Since this was used once to capture quay images for tests, I figured it belonged here